### PR TITLE
Add trunk add/remove network commands

### PR DESCRIPTION
**Pull request by tzumainn**
Adds additional CLI commands for adding and removing networks
from a trunk port.

### CODE REVIEW

File: esiclient/v1/trunk.py [hunk 1]
```diff
@@ -113,6 +114,125 @@ def take_action(self, parsed_args):
...
+        self.log.debug("take_action(%s)", parsed_args)
+
+        tagged_networks = parsed_args.tagged_networks
+
+        if len(tagged_networks) == 0:
+            raise exceptions.CommandError(
+                "ERROR: no networks specified")
+
+        neutron_client = self.app.client_manager.network
+        trunk = neutron_client.find_trunk(parsed_args.name)
```
Comment (larsks) [inline esiclient/v1/trunk.py:149]: What happens if this fails (e.g., operator provided an invalid trunk name on the command line)?

Comment (tzumainn) [reply esiclient/v1/trunk.py:149]: Hm, it returns a weird error message. I'll catch this case.

Note that adding a network to a trunk where that network is already attached will result in a neutron error that's passed down appropriately, so I don't handle those cases. I guess it's just find_* methods that can return None and mess things up.

File: esiclient/tests/v1/test_trunk.py [hunk 2]
```diff
@@ -328,3 +330,265 @@ def test_take_action(self):
...
+            "id": "network_uuid_2",
+            "name": "network2",
+            "provider_segmentation_id": 222
+        })
+        self.network3 = utils.create_mock_object({
+            "id": "network_uuid_3",
+            "name": "network3",
+            "provider_segmentation_id": 333
+        })
+        self.subport1 = utils.create_mock_object({
```
Comment (DanNiESh) [inline esiclient/tests/v1/test_trunk.py:351]: Maybe name the subports as subport2 and subport3 would align with the ports and networks id?

File: esiclient/tests/v1/test_trunk.py [hunk 3]
```diff
@@ -328,3 +330,265 @@ def test_take_action(self):
...
```
Comment (DanNiESh) [inline esiclient/tests/v1/test_trunk.py:481]: Same as above.

File: esiclient/tests/v1/test_trunk.py (hunk 2 above)
Comment (tzumainn) [reply esiclient/tests/v1/test_trunk.py]: Fair enough! Should be fixed now.

File: esiclient/tests/v1/test_trunk.py (hunk 3 above)
Comment (tzumainn) [reply esiclient/tests/v1/test_trunk.py]: Fixed as well.

File: esiclient/v1/trunk.py (hunk 1 above)
Comment (tzumainn) [reply esiclient/v1/trunk.py:149]: Okay, this should be fixed now!

**Comment by tzumainn** [comment]
Thanks for the reviews! I've fixed the issues. Let me know if there are any more!

File: esiclient/v1/trunk.py [hunk 4]
```diff
@@ -113,6 +114,133 @@ def take_action(self, parsed_args):
...
+
+        if trunk is None:
+            raise exceptions.CommandError(
+                "ERROR: no trunk named {0}".format(parsed_args.name))
+
+        sub_ports = []
+        for tagged_network_name in tagged_networks:
+            tagged_network = neutron_client.find_network(
```
Comment (larsks) [inline esiclient/v1/trunk.py:157]: Maybe do something similar here?

Comment (tzumainn) [reply esiclient/v1/trunk.py:157]: Oh, good point. Fixed!

**Pull request merged**